Source/Fuse.GeoLocation/LocationProvider.uno

```diff
@@ -83,6 +85,10 @@ namespace Fuse.GeoLocation
 			}
 		}
 
+		public bool IsLocationEnabled() {
```

Comment:
style: please put the opening curly on its own line

Reply:
will do...